src/clj/rems/db/test_data.clj

```diff
@@ -1086,7 +1087,15 @@
                              :resource-id res-organization-owner
                              :form-id form-organization-owner
                              :organization {:organization/id "organization1"}
-                             :workflow-id (:organization-owner workflows)})))
+                             :workflow-id (:organization-owner workflows)})
+    ;; create category
+
+    (db/create-category! {:id 89
```

Comment:
@Macroz I am experiencing some problems trying to add jsonb data into DB, looks like it does not get added into DB.

Reply:
Would you consider giving me a hand to debug it?

Reply:
How do you test it?

If I:
- pull latest
- start db and server (creating the test data as usual)
- look at generated data

```
select * from category;
 id |          data          
----+------------------------
 89 | {"name": "Category 1"}
 90 | {"name": "Category 2"}
 91 | {"name": "Category 3"}
(3 rows)
```

And that seems ok, doesn't it?

Reply:
Thank you for testing, it narrows down the problem

Reply:
fixed it, will push soon